sdk/storage/azure-storage-file-datalake/tests/test_file.py

```diff
@@ -98,10 +95,23 @@ def _create_file_and_return_client(self, directory="", file=None):
         file_client.create_file()
         return file_client
 
+    def _is_almost_equal(self, first, second, delta):
```

Comment:
Curious to see your feedback on this helper name. I avoided using `assert` since I don't want to mix up this class' helpers with any real `assert` functions from our test utils.

Reply:
Seems fine to me :)